azure.go

```diff
@@ -4,8 +4,8 @@ import (
 	"bytes"
 	"encoding/json"
 	"fmt"
+	"github.com/prometheus/common/log"
```

Comment:
github.com/prometheus/common/log is removed from the latest version of github.com/prometheus/common library

Reply:
it's promlog now

Reply:
nevermind